doc/en/fixture.rst

```diff
@@ -692,7 +692,7 @@ So let's just do another run:
     test_module.py:13: AssertionError
     ------------------------- Captured stdout teardown -------------------------
     finalizing <smtplib.SMTP object at 0xdeadbeef>
-    4 failed in 0.89s
+    4 failed in 1.69s
```

Comment:
this one still snuck through though :laughing:

Reply:
😢

Reply:
Unless there are further changes needed, I rather fix this in the next release. 😁

Reply:
sg2m